apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -318,8 +337,8 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
       )}
       <div>
         <Button
-          isDisabled={!isValid}
-          onClick={isValid ? handleNext : undefined}
+          isDisabled={!isValid || !passwordsMatch}
+          onClick={isValid && passwordsMatch ? handleNext : undefined}
```

Comment:
These should just be moved into the `isValid` declaration, that's why it exists as a single boolean

Reply:
done